nautobot/utilities/views.py

```diff
@@ -48,6 +48,25 @@ def dispatch(self, request, *args, **kwargs):
         return super().dispatch(request, *args, **kwargs)
 
 
+class AdminRequiredMixin(AccessMixin):
+    """
+    Allows access only to admin users.
+    """
+
+    def has_permission(self):
+        return bool(
```

Comment:
Since the nest logic is inherently a boolean, is there any chance that some weird behavior would cause this to fail open? Would/could this be better to do `...) or False` at the end?

Reply:
I don't think there would be a need for the `..) or False` because it would either return True or False